docs/operations/scalable-query-frontend.md

```diff
@@ -5,25 +5,52 @@ weight: 5
 slug: scaling-query-frontend
 ---
 
-Historically scaling the Cortex query frontend has [posed some challenges](https://cortexmetrics.io/docs/proposals/scalable-query-frontend/).  This document aims to detail how to use some of the added configuration parameters to correctly scale the frontend.  Note that these instructions apply in both the HA single binary scenario or microservices mode.
+Historically scaling the Cortex query frontend has [posed some challenges](../proposals/scalable-query-frontend.md).
+This document aims to detail how to use some added configuration parameters to correctly scale the frontend.
+Note that these instructions apply in both the HA single binary scenario or microservices mode.
 
-## DNS Configuration / Readiness
+## Scaling the Query Frontend
 
-When a new frontend is first created on scale up it will not immediately have queriers attached to it.  The existing endpoint `/ready` was updated to only return http 200 when the query frontend was ready to serve queries.  Make sure to configure this endpoint as a healthcheck in your load balancer.  Otherwise a query frontend scale up event might result in failed queries or high latency for a bit while queriers attach.
+For every query frontend the querier adds a [configurable number of goroutines](https://github.com/cortexproject/cortex/blob/1797adfed2979f6096c3305b0dc9162c1ec0c046/pkg/querier/worker/worker.go#L212)
```

Comment:
You know what I think about mentioning goroutines in the doc :) I don't think our doc should either talk about goroutines or link to code.